packages/server/test/unit/browsers/protocol_spec.ts

```diff
@@ -43,24 +43,44 @@ describe('lib/browsers/protocol', () => {
   })
 
   context('.getWsTargetFor', () => {
+    const expectedCdpFailedError = stripIndents`
+      Cypress failed to make a connection to the Chrome DevTools Protocol after retrying for 20 seconds.
```

Comment:
hardcoded time limit ;)

Reply:
indeed. now that foxdriver and marionette contain similar logic, i want to refactor this whole area of "connecting to browser-specific automation protocols"... but for now, this hard-coded time limit is still correct 🤷